01-node-tutorial/answers/09-path-module.js

```diff
@@ -0,0 +1,13 @@
+const path = require("path")
+
+console.log(path.sep)
+const filePath = path.join("/content//", "subfolder", "test.txt")
```

Comment:
`const filePath = path.join("content", "subfolder", "test.txt")
`

Reply:
I was trying it to see if it would ignore the backslash and I forgot to remove them :sweat_smile: